src/main/java/com/minecolonies/apiimp/initializer/ModTagsInitializer.java

```diff
@@ -0,0 +1,137 @@
+package com.minecolonies.apiimp.initializer;
+
+import com.minecolonies.api.items.ModTags;
+import com.minecolonies.api.util.constant.Constants;
+import net.minecraft.block.Block;
+import net.minecraft.item.Item;
+import net.minecraft.tags.BlockTags;
+import net.minecraft.tags.ItemTags;
+import net.minecraft.tags.Tag;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.event.TagsUpdatedEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.registries.ObjectHolder;
+import static com.minecolonies.api.util.constant.Constants.MOD_ID;
+import java.util.HashMap;
+
+@ObjectHolder(Constants.MOD_ID)
+@Mod.EventBusSubscriber(modid = Constants.MOD_ID, bus = Mod.EventBusSubscriber.Bus.FORGE)
+public class ModTagsInitializer
+{
+    /**
+     * The name of the tag (crafter) for improving recipes
+     */
+    private static final String REDUCEABLE = "reduceable";
+
+    /**
+     * Tag specifier for Products to Include
+     */
+    private static final String PRODUCT = "_product";
+
+    /**
+     * Tag specifier for Products to Exclude
+     */
+    private static final String PRODUCT_EXCLUDED = "_product_excluded";
+
+    /**
+     * Tag specifier for Ingredients to include
+     */
+    private static final String INGREDIENT = "_ingredient";
+
+    /**
+     * Tag specifier for Ingredients to exclude
+     */
+    private static final String INGREDIENT_EXCLUDED = "_ingredient_excluded";
+
+    private static final ResourceLocation DECORATION_ITEMS = new ResourceLocation(MOD_ID, "decoblocks");
+    private static final ResourceLocation CONCRETE_POWDER = new ResourceLocation(MOD_ID, "concrete_powder");
+    private static final ResourceLocation CONCRETE_BLOCK  = new ResourceLocation(MOD_ID, "concrete");
+    private static final ResourceLocation PATHING_BLOCKS = new ResourceLocation(MOD_ID, "pathblocks");
+
+
+    @SubscribeEvent
+    public static void tagsUpdated(TagsUpdatedEvent event)
+    {
+        ModTagsInitializer.init();
```

Comment:
Not needed.
We use named tags as such we do not need this.

Reply:
Apparently a static initializer is enough here.